cmd/cosign/cli/options/oidc.go

```diff
@@ -40,4 +41,7 @@ func (o *OIDCOptions) AddFlags(cmd *cobra.Command) {
 
 	cmd.Flags().StringVar(&o.ClientSecret, "oidc-client-secret", "",
 		"[EXPERIMENTAL] OIDC client secret for application")
+
+	cmd.Flags().StringVar(&o.RedirectURI, "oidc-redirect-uri", "",
+		"[EXPERIMENTAL] OIDC redirect URI")
```

Comment:
Can you note this is an optional flag, and note that its default behavior is to listen on localhost and a random port?